test/src/e2e/account.test.ts

```diff
@@ -210,4 +200,4 @@ describe("account", function() {
             await node.clean();
         });
     });
-});
+});
```

Comment:
It seems that your IDE or editor is not following the .editorconfig file.
Since in our `.editorconfig` file has the `insert_final_new_line = true` line in its file, this final new line should not be deleted.
https://github.com/CodeChain-io/foundry/blob/master/test/.editorconfig#L3

Please check this [link](https://github.com/kodebox-io/kodebox-internal/wiki/EditorConfig)